playbooks/remediate.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Remediate
+  hosts: all
+  strategy: free
+  become: true
+  force_handlers: true
+  vars:
+    remediation_todo:
+      - LEAPP_FIREWALLD_ALLOWZONEDRIFTING
+      - LEAPP_MISSING_PKG
```

Comment:
This will be taken from the variable in vars/main.yml right?

Reply:
I think yea, not sure how it should be defined or where when executing.

Reply:
The `playbooks` directory is linked to from the "Example playbooks" section of the main README file. These playbooks are there just to provide an example of how the role could be used. It is expected that role users will set vars as required in their own playbooks that include the role. As an example playbook, this one looks good to me.